packages/sdk/test/end-to-end/Operator.test.ts

```diff
@@ -177,6 +177,18 @@ describe('Operator', () => {
         }, 10000, 1000)
     }, 60 * 1000)  // TODO why this is slower, takes ~35 seconds?
 
+    it('avoids sending a transaction when trying to close non-existing flags', async () => {
+        const wallet = deployedOperator.nodeWallets[0] as Wallet
+        const operator = await getOperator(wallet, deployedOperator)
+        const nonceBefore = await wallet.getNonce() // nonce = "how many transactions sent so far"
+        await expect(operator.closeFlag(
+            toEthereumAddress(await sponsorship1.getAddress()),
+            toEthereumAddress(await deployedOperator.operatorContract.getAddress())
+        )).rejects.toThrowError()
```

Comment:
Better to wrap the promise creation to a function call so that we don't assume too much about the async implementation of `closeFlag()`. (This style doesn't catch the error if  the method would be a sync function which throws an exception instead of returning a `Promise`.)

```suggestion
        await expect(async () => operator.closeFlag(
            toEthereumAddress(await sponsorship1.getAddress()),
            toEthereumAddress(await deployedOperator.operatorContract.getAddress())
        )).rejects.toThrow()
```